x-pack/plugins/security_solution/public/resolver/models/location_search.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/**
+ * The legacy `crumbEvent` and `crumbId` parameters.
+ * @deprecated
+ */
+export function breadcrumbParameters(
```

Comment:
This file will have the logic for getting values out of (and constructing new) URLSearchParams